docs/application/native/guides/multimedia/overview.md

```diff
@@ -1,9 +1,9 @@
-# Media and Camera
+# Multimedia
```

Comment:
@sangchul1011
When we restructured the menu of "Media and Camera" guide, we divided "Media Content and Metadata" and "Audio Management ".
Now do you want to consolidate them?
If so, you have to update docs/application/native/guides/TOC.md.
AS-IS:
```
# Media and Camera
## [Overview](/application/native/guides/multimedia/overview.md)
## Media Content and Metadata
...
```

TO-BE:
```
# Multimedia
## [Overview](/application/native/guides/multimedia/overview.md)
## Media Content and Metadata
...
```

Reply:
I think it is not proper that the subject of /multimedia/overview.md is "Media and Camera". Simply we can change it to "Multimedia".

And honestly, I don't get it which standard is used to categorize for "Audio Management".
The previous "Audio Management" mentioned Sound-Manager, OpenAL and Sound-Pool. The latter two are for playback sound. So this is kind of ambiguity.
 
After this patch, we may raise another patch to re-align categories regarding audio APIs.

I'll update TOC.md file soon.